reV/config/collection.py

```diff
@@ -119,24 +100,52 @@ def _parse_pipeline_prefixes(self):
         return file_prefixes
 
     @property
-    def file_prefixes(self):
-        """Get the file prefixes to collect.
+    def fn_out_names(self):
+        """Get a list of output filenames ordered to correspond to the list of
+        collection patterns.
 
-        Returns
-        -------
-        file_prefixes : list
-            list of file prefixes to collect.
+        This can also be set to PIPELINE or just not set at all if this is a
+        pipeline job and collect_patterns="PIPELINE"
```

Comment:
I imagine the most typical use case will be not setting `fn_out_names` at all and `collect_patterns="PIPELINE"`, like you mention in the docstring here. This is exactly what I tried first, but it fails... If you follow the logic below for this case, you will hit the line `fn_out_names = list(fn_out_names)`, which gives you a `TypeError: 'NoneType' object is not iterable`